pkg/network/ebpf/c/protocols/kafka/kafka-parsing-helpers.h

```diff
@@ -0,0 +1,304 @@
+#ifndef __KAFKA_PARSING_HELPERS
```

Comment:
1. Indeed, I'm leaving it for later so I can focus on getting the version ready for the code freeze
2. Fixed

Reply:
that's super important, since most of the code here is a duplication, then:
1. You are adding a lot of complicated code, which led into the need to split the dispatcher
2. You are doing unnecessary work, instead of relying on the fact that we have a valid kafka message, you are revalidating the packet, instead of reading the relevant fields
3. more code, means that we need to go through the changes and verify them, possible more comments -> longer CR cycle

Reply:
Changed